features/step_definitions/purchase_steps.rb

```diff
@@ -93,11 +91,12 @@
 end
 
 Then(/^my credit card is saved for future purchases$/) do
-  expect(@user.card_token).to_not be_nil
+  sleep(5)
```

Comment:
@dkniffin I'm trying to give Stripe a few seconds to process the order. Without it, my tests move on to the next step before the order is processed and the confirmation steps fail. Thoughts on a better way to do this?